server/pulp/plugins/rsync/publish.py

```diff
@@ -258,6 +258,36 @@ def process_main(self):
             raise PulpCodedException(message=output)
 
 
+class UpdateLastPredistDateStep(PublishStep):
```

Comment:
I do think it should be left where it is.  Considering that it's not used just by the rsync distributor but also the ISO rsync distributor, there's not an obvious place to put it at a lower level.